src/notation/internal/notationinteraction.cpp

```diff
@@ -731,6 +731,107 @@ void NotationInteraction::selectTopOrBottomOfChord(MoveDirection d)
     showItem(target);
 }
 
+static bool elementLower(const EngravingItem* e1, const EngravingItem* e2)
```

Comment:
I see a big similarity between this method and `NotationInteraction::elementIsLess`. Maybe the latter can be used instead?